pkg/interactive/validation.go

```diff
@@ -139,9 +139,13 @@ func RegExpBoolean(restr string) Validator {
 
 // SubnetsCountValidator get a slice of `[]core.OptionAnswer` as an interface.
 // e.g. core.OptionAnswer { Value: subnet-04f67939f44a97dbe (us-west-2b), Index: 0 }
-func SubnetsCountValidator(multiAZ bool, privateLink bool) Validator {
+func SubnetsCountValidator(multiAZ bool, privateLink bool, hostedCP bool) Validator {
 	return func(input interface{}) error {
 		if answers, ok := input.([]core.OptionAnswer); ok {
+			if hostedCP {
```

Comment:
What is the minimal number of subnets for a Hypershift cluster?

Reply:
https://github.com/openshift/rosa/blob/master/pkg/ocm/helpers.go#L641

Reply:
The minimal number is 1 in private and 2 in public (singleaz/multiaz doesn't matter for hosted clusters).

We can add that validation there, but after out of this block we need to query AWS to check public/private subnets, and the private number is collected for further replicas validation.

So, +1 of adding the basic number in that block, but I didn't see fit to query AWS from it.

Reply:
I agree, just the minimal validation, to avoid this:
![image](https://user-images.githubusercontent.com/57869309/226904699-0fc47c08-b16b-46e0-8a69-917317e117b5.png)
So we can give the user the opportunity to select more/fewer subnets instead of starting from the beginning again.

Reply:
Yeah, added.
Perhaps I was biased, as the same validation was done in the "ValidateHostedClusterSubnets".